test/shipping_zone_test.js

```diff
@@ -0,0 +1,28 @@
+var helper = require("./common.js");
+helper.setObject("shipping_zone");
+
+var Resource = helper.resource();
+
+helper.nock(helper.test_shop)
+  .get('/admin/shipping_zones.json')
+  .reply(200, helper.load("all"), {
```

Comment:
Save the fixture somewhere, so we can reuse it later and remove the third argument.